src/main/scala/chisel3/util/SRAM.scala

```diff
@@ -261,6 +349,9 @@ object SRAM {
       )
     }
 
+    // Emit Verilog for preloading the memory from a file if requested
+    memoryFile.map { file: MemoryFile => loadMemoryFromFileInline(mem, file.path, file.fileType) }
```

Comment:
```suggestion
    memoryFile.foreach { file: MemoryFile => loadMemoryFromFileInline(mem, file.path, file.fileType) }
```
Use foreach when return value is `Unit` or we otherwise don't care about the return value